code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -375,16 +375,16 @@
 	var/T1 = created_volume * 20		//100 units : Zap 3 times, with powers 2000/5000/12000. Tesla revolvers have a power of 10000 for comparison.
 	var/T2 = created_volume * 50
 	var/T3 = created_volume * 120
-	sleep(10)
-	if(created_volume >= 75)			//10 units minimum for lightning, 40 units for secondary blast, 75 units for tertiary blast.
+	sleep(5)
+	if(created_volume >= 75)
 		tesla_zap(holder.my_atom, 7, T1)
 		playsound(holder.my_atom, 'sound/machines/defib_zap.ogg', 50, 1)
-	sleep(10)
+		sleep(15)
```

Comment:
so you don't have to wait 3.5 seconds for it to explode if you have less than 75 teslium